src/api/app/presenters/notification_presenter.rb

```diff
@@ -40,13 +40,18 @@ def excerpt
     text.to_s.truncate(100)
   end
 
-  def kind_of_request
-    return unless @model.notifiable_type == 'BsRequest'
+  def for_request?
+    @model.notifiable_type == 'BsRequest'
+  end
 
+  def request_actions
     request = @model.notifiable
-    return "Multiple actions for project #{request.bs_request_actions.first.target_project}" if request.bs_request_actions.size > 1
+    request_action = request.bs_request_actions.first
+    return "Multiple actions for project #{request_action.target_project}" if request.bs_request_actions.size > 1
 
-    BsRequest.actions_summary(@model.event_payload)
+    # This is to avoid relying on bs_request.webui_actions, which is a method we need to remove
+    request_action_hash = request_action.serializable_hash.merge(type: request_action.type).with_indifferent_access
+    Webui::WebuiController.helpers.request_action_header(request_action_hash, request.creator)
```

Comment:
It's quite inconsistent how we display requests and their actions throughout the application. For example, in the project#pulse page, we have an icon for each request state. We don't do that anywhere else. `BsRequest.actions_summary` is also used to describe actions, but only in events (so for `app/models/event/`). In the end, I went with the helper method `request_action_header` since it's the one we use on request#show.